### ISSUE TITLE
Bug: Linked Listing items aren't recognized as links

### ISSUE THREAD

**Issue by filwi** [CrocoBlock/suggestions#6178]
**Describe the bug**
When creating a listing where the whole item is made clickable by the "Make listing item clickable"-function under listing settings, the event listener set to listen to a click on the item in the listing grid doesn't announce itself as a link.

This means that web browser link functions, such as "Open in new tab" or "Open in new page" aren't available for whole list item links:

![MissingLink](https://user-images.githubusercontent.com/8066290/202434119-5e59f852-8f7e-49f1-a00c-f362f65c75ad.jpg)

**To Reproduce**
1. Create listing item.
2. Set "Make listing item clickable" to "yes"
3. Save listing item.
4. Add listing item to listing grid.
5. Publish page.
6. It is now possible to go to the link in the listing item by clicking on it, but not to open it in a new page or tab.

**Expected behavior**
Listing grid items should behave just like regular links - with the ability to, for example, Ctrl-click and open the link in a new page.

**Desktop (please complete the following information):**
- JetEngine version: 3.0.6.1
- Elementor version: 3.8.1
- Wordpress version: 6.1
- OS: Windows 10 Pro v 21H2
- Browser: Firefox v106.0.5 (64-bit)

**Comment by Emilracerbil** [comment]
Yes i am struggling with this too. It is also pretty much impossible to copy specific text from a listing item when it is set as clickable.

I found a workaround code snippet that makes the entire listing possible to open in a new tab, however, its still impossible to copy specific text from the listing.

```

.jet-engine-listing-overlay-link {
	z-index:1;
}

.jet-listing-grid .jet-engine-listing-overlay-link {
	z-index: 1000;
}

.hovered {
	-moz-transform: scale(1.1);
	-webkit-transform: scale(1.1);
	transform: scale(1.1);
}

```

**Comment by morosmo** [comment]
@Emilracerbil 
Thankyou for Code, Hope team will fix it soon

**Comment by morosmo** [comment]
+1
